web_common/web_common/styles/main.scss

```diff
@@ -224,15 +224,14 @@ a {
 
 .data-table {
     min-width: 600px;
-    border-spacing: 2px;
-    thead {
-	color: $white;
-	background-color: #888;
-    }
+
     th {
         white-space: nowrap;
-	padding: 3px 10px;
+        padding: 3px 10px;
+        color: $white;
+        background-color: #888;
```

Comment:
Specifying background color on the th's is more reliable. You can test this by having a table overflow a container: if background is set on the thead, the overflowed cells will not contain the background on scroll. 

Also semantically a bit clearer I think: the thead doesn't actually take on the background color when used as a table, in that the spaces between th's have no color.